src/components/tasks/Monday.tsx

```diff
@@ -279,52 +286,73 @@ const Monday = (props: MondayProps) => {
         }}
         {...register("element.description")}
       />
-      <TextField
-        select
-        label="Select Workspace"
-        required
-        value={selectedWorkspace}
-        onChange={(e) => {
-          setSelectedWorkspace(e.target.value);
-        }}
-      >
-        {mondayWorkspacesData?.mondayWorkspaces.elements?.map(
-          (mondayWorkspaceElement) => (
-            <MenuItem
-              key={mondayWorkspaceElement.id}
-              value={mondayWorkspaceElement.id}
-            >
-              {mondayWorkspaceElement.name}
-            </MenuItem>
-          ),
-        ) ?? (
-          <MenuItem disabled>
-            There are no workspace available to select
-          </MenuItem>
+      <Controller
+        render={({ field: { onChange, ...rest } }) => (
+          <TextField
+            select
+            label="Select Workspace"
+            required
+            onChange={(e) => {
+              resetField("element.group", { defaultValue: null });
+              onChange(e.target.value);
+            }}
+            {...rest}
```

Comment:
same here

Reply:
Actually, kind of same remarks for all {...rest} ^^

Reply:
I'll consider it okay since it's inside a controller and gets warned by the IDE